docs/content/workflows/pipelines/actions/builtin/gitclone.md

```diff
@@ -8,10 +8,6 @@ chapter = true
 
 This action clones a repository into a new directory.
 
-This will retrieve a shallow git clone (`depth==1`).
-
-You may want to use a privateKey to clone from an SSH repository. To do so, you will need to add a project or an application variable of type `key`. `{{.cds.app.a-key.pub}}`
-
 ## Parameters
```

Comment:
maybe add a note to indicate that if you don't give url it will check in the VCS application repo strategy ?

Reply:
done